src/main.py

```diff
@@ -308,8 +308,12 @@ def main():
 
         # Changed this logic to check only for OPEN PRs for dev purposes
         if pr_status == "OPEN":
-            log(f"Skipping vulnerability {vuln_uuid} as an OPEN or MERGED PR with label '{label_name}' already exists.")
+            log(f"Skipping vulnerability {vuln_uuid} as an OPEN PR with label '{label_name}' already exists.")
             log("\n::endgroup::")
+            if vuln_uuid in skipped_vulns:
+                log(f"Already skipped {vuln_uuid} before, breaking loop to avoid infinite loop.")
```

Comment:
What is the loop here? get_vulnerability_with_prompts() starts returning the same vulnerability uuid multiple times?   Should it take some sort of payload so that it knows what vulnerabilities should not be returned?